html-forms/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="http://127.0.0.1:5500/html-forms/index.html">
+      <label>
+        <div>Email</div>
+        <input type="email" name="Email" id="email" />
+      </label>
+    </form>
+    <form action="http://127.0.0.1:5500/html-forms/index.html">
+      <div>
+        <label for="why is coding cool">Why is coding cool</label>
+      </div>
+      <select name="Why is coding cool" id="why is coding cool">
+        <option value="Select and option">Select an Option</option>
+        <option value="Because it's interesting">
+          Because it's interesting
+        </option>
+        <option value="Because it's fun">Because it's fun</option>
+        <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+    </form>
+    <form action="http://127.0.0.1:5500/html-forms/index.html">
+      <div>
+        <label for="Favorite Language (choose one)"
+          >Favorite Language (choose one)</label
+        >
+      </div>
+      <input type="radio" name="Language" value="HTML" />
+      <label for="HTML">HTML</label>
+      <input type="radio" name="Language" value="CSS" />
+      <label for="CSS">CSS</label>
+      <input type="radio" name="Language" value="Javascript" />
+      <label for="Javascript">Javascript</label>
+    </form>
+    <form action="http://127.0.0.1:5500/html-forms/index.html">
+      <div>
+        <label for="Would you like to sign up for more information?"
+          >Would you like to sign up for more information?</label
+        >
+      </div>
+      <input
+        type="checkbox"
+        name="More Information"
+        value="Sign me Up"
+        id="Sign Up" />
+      <label for="Sign Me Up">Sign Me Up!</label>
+    </form>
+    <form action="http://127.0.0.1:5500/html-forms/index.html">
+      <div>
+        <label for="comments">Comments</label>
+      </div>
+      <textarea name="Comments" cols="20" rows="7" id="Comments"> </textarea>
```

Comment:
watch out for the capital `c` in comments! it doesn't match the `c` in the label.